src/main/java/jenkins/plugins/git/GitSCMSourceDefaults.java

```diff
@@ -88,7 +88,7 @@ public boolean equals(Object o) {
      */
     @Override
     public int hashCode() {
-        return (includeTags ? 1 : 0);
+        return includeTags ? 1 : 0;
```

Comment:
Please remove this change.  It is a change in the style of whitespace changes, modifying a line in a way that risks conflicts on other pull requests.

Reply:
done